### ISSUE TITLE
Windows project is set as Unpackaged by default

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno.templates#392]
## What would you like to be added:

Since the last release, Windows project defaults to launch as Unpackaged. This can be problematic as some of the features we provide in cross-platform way are not working in unpackaged Windows apps currently - e.g. `ApplicationData`. Is there a reason why Unpackaged is the default now?

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

**Comment by MartinZikmund** [comment]
CC @nickrandolph you probably know why this was done?

**Comment by nickrandolph** [comment]
Yes, done to enable HR

**Comment by nickrandolph** [comment]
@MartinZikmund are there open issues on uno that we can pick up to address any issues with unpackaged?
We have some (eg https://github.com/unoplatform/uno.extensions/issues/1370) in extensions for this

**Comment by nickrandolph** [comment]
cc @jeromelaban thoughts on leaving the default as packaged or unpackaged?

**Comment by jeromelaban** [comment]
The issue about unpackaged/packaged is mostly about hot reload, right? The others are related to resources working in one mode and not in the other.

**Comment by nickrandolph** [comment]
Yes, so we set to unpackaged to give best developer experience w.r.t. hot reload
Can switch it back if we think that the applicationdata issue is too much of a show stopper?

**Comment by MartinZikmund** [comment]
Probably OK to keep it this way, unless people start complaining, but I think it should be mentioned in the docs for Application Data that on Windows it will work only in case of Packaged app
